components/OwnDetailPage/index.js

```diff
@@ -67,9 +68,17 @@ const OwnRecipeDetails = () => {
       )}
 
       <form onSubmit={handleSaveNote}>
-        <StyledTextarea value={note} name="notes" onChange={handleChange} />
-        <br /> <br />
+        <StyledTextarea
```

Comment:
Since the `textarea` element doesn't have a label associated with it you should add an `aria-label` attribute. The placeholder is not as reliable with screen readers as a label.